CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -342,21 +342,18 @@ public function postProcess() {
   /**
    * Based on input, create participant row for transferee and send email
    *
-   * return @ void
+   * @param $participant
    *
+   * @throws \API_Exception
    * @throws \CRM_Core_Exception
+   * @throws \CiviCRM_API3_Exception
    */
-  public function participantTransfer($participant) {
-    $contactDetails = [];
-    $contactIds[] = $participant->contact_id;
-    list($currentContactDetails) = CRM_Utils_Token::getTokenDetails($contactIds, NULL,
-      FALSE, FALSE, NULL, [], 'CRM_Event_BAO_Participant');
```

Comment:
It seems significant to me that (1) old code called `CRM_Utils_Token::getTokenDetails()` but didn't do anything to indicate `returnProperties` or the active-token-list. So it'd always return the same fields. (2) The subsequent references to `$contactDetails` only u se `display_name` and `email`.

So the change here looks like a proper simplification.